docs/389ds/FAQ/esc-guide.md

```diff
@@ -27,7 +27,7 @@ The Dogtag Certificate System creates, manages, renews, and deletes certificates
 -   The Data Recovery Manager (DRM), which archives and recovers keys.
 -   The Online Certificate Status Manager, which stores lists of revoked certificates for client applications to use to check if a certificate is valid.
 -   The Token Processing System (TPS), which interacts with smart cards to generate and store keys and certificates for a specific user.
--   The Token Key Service (TKS), which generates and stores master keys used by the TPS.
```

Comment:
Are you sure we can change this word. Someone from PKI should confirm

Reply:
I've changed it back as I confirmed with the recent docs that they still use 'master' terminology.